dWorldServer/WorldServer.cpp

```diff
@@ -1244,7 +1245,11 @@ void HandlePacket(Packet* packet) {
 	}
 
 	default:
-		LOG("Unknown world packet received: %i", int(packet->data[3]));
+		if (packet->bitSize < (16 + sizeof(uint32_t) * 8)) break;
+
+		uint32_t messageId = *reinterpret_cast<uint32_t*>(&packet->data[3]);
```

Comment:
This should be able to just be reinterpret_cast straight to eWorldMessageType, can it not? You could also inline these two to be `const char* messageIdString = StringifiedEnum::ToString(*reinterpret_cast<eWorldMessageType*>(&packet->data[3])).data();`